corehq/apps/linked_domain/models.py

```diff
@@ -101,7 +101,8 @@ def link_domains(cls, linked_domain, master_domain, remote_details=None):
         if active_links_with_other_domains:
             already_linked_domain = active_links_with_other_domains[0].master_domain
             raise DomainLinkError(
-                _(f'{linked_domain} is already a downstream project space of {already_linked_domain}.')
+                _(f'{linked_domain} is already a downstream project space of {already_linked_domain}.\n'
+                  'You must remove the existing link before creating this new link.')
```

Comment:
Why would someone want/need to re-create a link?

Reply:
This is preventing a multi-upstream case. The `active_links_with_other_domains` is a list of other links where the `linked_domain` is a downstream domain.